src/GraphQL.Tests/Utilities/SchemaPrinterTests.cs

```diff
@@ -798,7 +798,7 @@ public void prints_enum_default_args()
                 Arguments = new QueryArguments(new QueryArgument<RgbEnum>
                 {
                     Name = "color",
-                    DefaultValue = "RED"
+                    DefaultValue = 0 // 0 = red --- must be internal representation of enumeration value or validation will fail
                 }),
```

Comment:
Internal representations of enum values must be respected for `DefaultValue`.  So setting it to "RED" is invalid because that is an external representation, not an internal representation.  I would almost say this was a bug, as it certainly can cause hidden problems in users' code.

Reply:
OK. Can it be `Color.Red` ?

Reply:
`Color.Red` is more native for me even than `0`. Need a test.

Reply:
There is an overabundance of tests for enums located in `Bug1699InvalidEnum`.  We explicitly test for both - see the `Simple_Enum` and `Enum_From_Number` tests.

However, in this particular test, the enumeration is created by the schema builder, so there is no native .NET enum type.

Reply:
OK